### PR TITLE
BOOK Model added

### PR DESCRIPTION
**Pull request by ramrap**

### CODE REVIEW

File: bookApp/models.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+# Create your models here.
+class keywords(models.Model):
```
Comment (subodhk01) [inline bookApp/models.py:5]: Model name should start with capital and should be singular

File: bookApp/models.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        ('B','Both')
+    )
+    price=models.IntegerField(blank=False,default=0)
+    #Borrowing method
+    bMethod=models.CharField(max_length=1,choices=borrowing_choice)
+
+    #status True means availabe
+    borrowedBy=models.ForeignKey(User, on_delete=models.CASCADE,related_name='+',null=True)
+    status=models.BooleanField(default=True)
+    publication_date=models.DateField(null=True, auto_now=False)
```
Comment (subodhk01) [inline bookApp/models.py:28]: Exact date of publication is not required, only the year can suffice.

File: bookApp/models.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    title=models.TextField(max_length=50,
+        blank=False,
+        unique=False)
+    author=models.TextField(max_length=32,blank=False)
+    owner=models.ForeignKey(User, on_delete=models.CASCADE)
+    description=models.TextField(max_length=512,blank=True)
+    borrowing_choice=(
+        ('R','Rent'),
+        ('S','Sell'),
+        ('B','Both')
```
Comment (subodhk01) [inline bookApp/models.py:19]: what does 'Both' represent ??

File: bookApp/models.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    )
+    price=models.IntegerField(blank=False,default=0)
+    #Borrowing method
+    bMethod=models.CharField(max_length=1,choices=borrowing_choice)
+
+    #status True means availabe
+    borrowedBy=models.ForeignKey(User, on_delete=models.CASCADE,related_name='+',null=True)
+    status=models.BooleanField(default=True)
+    publication_date=models.DateField(null=True, auto_now=False)
+    cover=models.ImageField(blank=None, upload_to='covers/', height_field=None, width_field=None, max_length=None)
```
Comment (subodhk01) [inline bookApp/models.py:29]: the the folder name to something like "book/covers"

File: bookApp/models.py [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    keyword=models.ManyToManyField(keywords)
```
Comment (subodhk01) [inline bookApp/models.py:30]: There might be multiple keywords so name it as "keywords"

**Review by subodhk01 — CHANGES_REQUESTED** [review]
And also arrange the models fields so as the similar fields remain close to each other.
Also your PR is trying to delete USER_MODEL.md and trying to do some changes in BOOK_MODEL.md, fix that also.

File: bookApp/models.py (hunk 1 above)
Comment (ramrap) [reply bookApp/models.py:5]: DOnne

File: bookApp/models.py (hunk 2 above)
Comment (ramrap) [reply bookApp/models.py:28]: Done

File: bookApp/models.py (hunk 3 above)
Comment (ramrap) [reply bookApp/models.py:19]: use your own mind

File: bookApp/models.py (hunk 4 above)
Comment (ramrap) [reply bookApp/models.py:29]: done

File: bookApp/models.py (hunk 5 above)
Comment (ramrap) [reply bookApp/models.py:30]: done

**Pull request closed without merging**